### PR TITLE
docu: update url for ms teams notifications, fixes #362

### LINKED ISSUE #362: Adjust kured to use the new Microsoft Teams connector webhook URL format
As mentioned in [#138 of shoutrrr](https://github.com/containrrr/shoutrrr/issues/138) the Microsoft Teams connector webhook URL format changed. Before it was `https://outlook.office.com/webhook/{tokenA}/IncomingWebhook/{tokenB}/{tokenC}` wheras of today, the base url is `https://<tName>/....`

Thus one should be able to also specify the `<tName>` i.e. something like this: `teams: teams://tName/token-a/token-b/token-c`

The new url is already reflected in v0.4.0 of `shoutrrr`

### PR DESCRIPTION
**Pull request by papanito**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -103,7 +106,7 @@ Flags:
       --reboot-sentinel string              path to file whose existence signals need to reboot (default "/var/run/reboot-required")
       --reboot-sentinel-command string      command for which a successful run signals need to reboot (default ""). If non-empty, sentinel file will be ignored.
       --slack-channel string                slack channel for reboot notfications
-      --slack-hook-url string               slack hook URL for reboot notfications
+      --slack-hook-url string               slack hook URL for reboot notfications [depracated in favor of --notify-url]
```
Comment (evrardjp) [inline README.md:109]: I think it's spelled "deprecated"

File: README.md [hunk 2]
```diff
@@ -1,27 +1,29 @@
 
-# kured - Kubernetes Reboot Daemon
+# kured - Kubernetes Reboot Daemon  <!-- omit in toc -->
```
Comment (evrardjp) [inline README.md:2]: Not sure we need this, and if we do, it should be a separate commit explaining why we do this.

File: README.md [hunk 3]
```diff
@@ -243,7 +246,7 @@ slack:           `slack://tokenA/tokenB/tokenC`
 
 rocketchat:      `rocketchat://[username@]rocketchat-host/token[/channel|@recipient]`
 
-teams:           `teams://token-a/token-b/token-c`
+teams:           `teams://tName/token-a/token-b/token-c`
```
Comment (evrardjp) [inline README.md:249]: Will that be clear enough, or should we link to a a reference explaining how to get tName?

Comment (papanito) [reply README.md]: Added some explanation and a link

**Review by ckotzbauer — APPROVED** [review]

**Comment by papanito** [comment]
thanks @evrardjp and @ckotzbauer

**Comment by papanito** [comment]
ehm, I guess somebody has to merge, I don't have the permissions to do so ;-)

**Comment by ckotzbauer** [comment]
@evrardjp Do you see any further todos here before merge?

File: README.md [hunk 4]
```diff
@@ -3,25 +3,27 @@
...
+  - [Blocking Reboots via Pods](#blocking-reboots-via-pods)
+  - [Prometheus Metrics](#prometheus-metrics)
+  - [Notifications](#notifications)
+  - [Overriding Lock Configuration](#overriding-lock-configuration)
+- [Operation](#operation)
+  - [Testing](#testing)
+  - [Disabling Reboots](#disabling-reboots)
+  - [Manual Unlock](#manual-unlock)
+  - [Automatic Unlock](#automatic-unlock)
+  - [Delaying Lock Release](#delaying-lock-release)
```
Comment (evrardjp) [inline README.md:22]: This is not really part of this PR, but I think it's fine

**Review by evrardjp — APPROVED** [review]

**Comment by papanito** [comment]
Thanks!
